.eslintrc.js

```diff
@@ -75,12 +75,68 @@ module.exports = {
     ],
     'no-plusplus': ['error', { allowForLoopAfterthoughts: true }],
   },
-
   globals: {
     __DEVELOPMENT__: true,
     __CLIENT__: true,
     __SERVER__: true,
     __DISABLE_SSR__: true,
     __DEVTOOLS__: true,
   },
+  overrides: [
+    // typescript .d.ts config
+    {
+      extends: [
+        'eslint:recommended',
+        'plugin:@typescript-eslint/eslint-recommended',
+        'plugin:@typescript-eslint/recommended',
+      ],
```

Comment:
I mentioned before https://github.com/patw0929/react-intl-tel-input/pull/351#issuecomment-646131706 that I couldn't lint `.d.ts` files. Now, I can.